spec/support/helpers/configuration_helper.rb

```diff
@@ -3,36 +3,74 @@
 require 'dotenv'
 require 'jwt'
 
-Dotenv.load('.env.test')
+DOTENV_FILE_NAME = '.env.test'
+Dotenv.load(DOTENV_FILE_NAME)
+
+REFRESH_TOKENS = ENV.fetch('REFRESH_TOKENS')
```

Comment:
Added to support using a real token, for instance when rerecording VCR cassettes as I were doing now.